package.json

```diff
@@ -33,7 +33,7 @@
     "logo": "https://opencollective.com/react-native-elements/logo.txt"
   },
   "dependencies": {
-    "@types/react-native-vector-icons": "^6.4.5",
+    "@types/react-native-vector-icons": "^6.4.6",
```

Comment:
I think we should move this to `devDeps`

Reply:
I attempted this before. However, #2423 happened when I did this. I'm sure there's a better answer, but this may need to be handled in a separate ticket.